moviecenter/app.py

```diff
@@ -0,0 +1,29 @@
+from fastapi import APIRouter, HTTPException, Request
+
+from .movies import Movie, add_movie, delete_movie, get_all_movies, get_movie_by_name
+
+movies_router = APIRouter()
+
+
+@movies_router.get("/movies", response_model=list[Movie])
+async def get_all_movies_api(request: Request) -> list[Movie]:
+    return await get_all_movies(request.app.state.db)
```

Comment:
Is this typed? Does mypy know `request.app.state.db` is a DB connection?
If not. consider wrapping it with a casting-only callabale and `fastapi`'s `Depends` to clarify